cmd/conformance/gcp/main.go

```diff
@@ -104,8 +110,39 @@ func main() {
 			_, _ = w.Write([]byte(err.Error()))
 			return
 		}
+
+		// If it's a dup, read the entry out from the bundle to simulate what CT could do in this case.
+		if idx.IsDup {
+			_, cpRaw, err := await.Await(ctx, f)
+			if err != nil {
+				w.WriteHeader(http.StatusInternalServerError)
+				_, _ = w.Write([]byte(err.Error()))
+				return
+			}
+			bits := strings.Split(string(cpRaw), "\n")
+			if len(bits) < 3 {
+				w.WriteHeader(http.StatusInternalServerError)
+				_, _ = w.Write([]byte("invalid checkpoint"))
+				return
+			}
+			treeSize, err := strconv.ParseUint(bits[1], 10, 64)
+			if err != nil {
+				w.WriteHeader(http.StatusInternalServerError)
+				_, _ = w.Write([]byte(err.Error()))
+				return
+			}
```

Comment:
Consider doing this properly. This encourages people to do unsafe things with checkpoints.